src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -133,6 +133,11 @@ public void periodic() {
         double positionOfArm = armPositionEncoder.getAbsolutePosition().getValue() * 360;
         double pivotError = pivotMotor.getClosedLoopError().getValue() * 360;
 
+        String SMART_DASHBOARD_ISNOTEINSHOOTER = "Note in Shooter";
+        SmartDashboard.putBoolean(SMART_DASHBOARD_ISNOTEINSHOOTER, isNoteInShooter());
+        String SMART_DASHBOARD_ISREADYTOFIRE = "Ready To Fire";
+        SmartDashboard.putBoolean(SMART_DASHBOARD_ISREADYTOFIRE, isNoteReadyToFire());
```

Comment:
If you're only using the constants in one location, you don't really need to make them a separate variable.